jobs/notebooks/monitor_presence_model.scala

```diff
@@ -0,0 +1,62 @@
+// Databricks notebook source
```

Comment:
All of this notebook is Python code, but it's still imported as a Scala notebook. Wouldn't it make more sense to convert it to a Python notebook? Then we can get rid of all the `//MAGIC %python`